sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1110,6 +1110,16 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
     }
     assert(e.getMessage.contains("Cannot create encoder for Option of Product type"))
   }
+
+
+  test ("SPARK-17460: If the sizeInBytes in Statistics exceeds the limit of an Int, " +
```

Comment:
You do not need to use such a long test case name. Just put the description inside the test case.